index.html

```diff
@@ -165,7 +177,17 @@ <h1 class="main-header hide-under-1024" data-caption-animate="fadeInUp">It's tim
 					</div>
 					<div class="row">
 						<div class="col-md-8 col-md-offset-2 col-sm-8 col-sm-offset-2 col-xs-12 text-center">
-							<p>SeekHealing is a detox aftercare model that unlocks the neurochemical benefits of authentic connection experiences. This non-profit organization provides equal-opportunity programs that use principles of connection & community to facilitate sustainable healing after detox from opioid &amp; heroin addiction.</p>
+							<p class="font-18">A detox aftercare model that unlocks the neurochemical benefits of authentic connection experiences. This non-profit organization provides equal-opportunity programs that use principles of connection & community to empower sustainable healing after detox from opioid & heroin addiction.</p>
+						</div>
+					</div>
+					<div class="heading-block center"></div>
+					<div class="row">
+						<div class="col-md-8 col-md-offset-2 col-sm-8 col-sm-offset-2 col-xs-12 text-center">
+							<p>Founded in 2016, SeekHealing is a 501(c)3 charity and post-detox recovery model that challenges mainstream stigmas about opioid and heroin addiction. Our programs turn the idea of standard rehab upside down by providing affordable services that prioritize authentic connection and harm reduction.</p>
+
+							<p>Our programs are designed for the 6-12 months after detox that are most critical to sustainable recovery and mental health integration. The methodology is focused to enhance the neuroplasticity of participants through: intentional experiences of authentic social & spiritual connection; rewarding the practice of learning new things; and facilitating a major change to their daily environment.</p>
```

Comment:
Sorry - same HTML entity comment. 🙄 Thanks for accommodating my pedantry. There's one two lines later too.

Reply:
No worries at all, I converted all the Ampersands now in b2321aa